frontend/app/dashboard/src/views/dashboard/groups/CategoryView.vue

```diff
@@ -170,6 +195,16 @@ export default class CategoryView extends Mixins(NavigationMixin) {
         }))
     }
 
+    showContextMenu(event,group: Group) {
+      const displayedComponent = new ComponentWithProperties(GroupContextMenu, {
+        x: event.clientX,
+        y: event.clientY + 10,
+        group: group,
+        cycleOffset: 0,
```

Comment:
what is the `cycleOffset` used for? I don't think we need it in the GroupContextMenu

Reply:
I don't think you need it here. This only has a meaning in a different context. cycleOffset represents the difference in the cycle of a group. Every year/month/... you can start a new registration period = cycle. For every registration, we keep track of the current cycle that was registered for.

Reply:
There is no such property in GroupContextMenu, so no reason to pass it
![image](https://user-images.githubusercontent.com/5277847/123069403-09dede00-d413-11eb-9cd0-51507c69dd24.png)